config/schema/belgie/order/postnl/nl_package.json

```diff
@@ -8,5 +8,9 @@
         }
       }
     }
+  },
+  "shipmentOptions": {
```

Comment:
Volgens mij is dit blokje `shipmentOptions` niet nodig want dezelfde inhoud staat al in de base schema `order.json`, toch?

Reply:
Ja, je hebt gelijk. Ik ga het aanpassen

> Volgens mij is dit blokje `shipmentOptions` niet nodig want dezelfde inhoud staat al in de base schema `order.json`, toch?